games-misc/lolcat/lolcat-42.24.0.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
```

Comment:
- bump to `EAPI=6`
- add all modern ruby impls that work please, and remove old ones